src/3.x/package-development.md

```diff
@@ -38,28 +38,28 @@ Event | Explanation
 `checkout-payment-saved` | After the payment method is saved
 `product-super-attribute-change` | After a swatch change, when calling this the product image updates based on the choice
 
-There is also a `doNotGoToTheNextStep` variable on the root Vue instance which can be used to prevent the checkout from going to the next step. That's also used within the [Rapidez Mollie](https://github.com/rapidez/mollie) package to prevent the checkout from going to the success page because you've to pay first and we'd like to redirect the user to the payment page.
+There is also a `doNotGoToTheNextStep` variable on the root Vue instance which can be used to prevent the checkout from going to the next step. That's also used within the [Rapidez Mollie](https://github.com/rapidez/mollie) package to prevent the checkout from going to the success page because you have to pay first and we'd like to redirect the user to the payment page.
 
 ## Fallback routing
 
-If your package cannot define it's own predefined routes you will want to start using fallback routes to check if it matches a route in e.g. your database.
+If your package cannot define its own predefined routes you will want to start using fallback routes to check if it matches a route in e.g. your database.
 
 ### `Rapidez::addFallbackRoute()`
 
 ::: info Why?
 If you use the `Route::fallback()` you'll prevent other packages from implementing fallback routes, this is what we've created `Rapidez::addFallbackRoute()` for.
 :::
 
-You can pass controllers, functions etc. in the same way like you would with `Route::fallback()` however it will check each fallback route until it finds one that does not return void or a 404. You can use this function anywhere so long as it's before the fallback route is triggered, we suggest in your ServiceProvider or Routes file ([example](https://github.com/rapidez/core/blob/aa1dbb54faed244b982f5b6198749ccf493c210a/src/RapidezServiceProvider.php#L87)).
+You can pass controllers, functions, etc. in the same way like you would with `Route::fallback()` however it will check each fallback route until it finds one that does not return void or a 404. You can use this function anywhere so long as it's before the fallback route is triggered, we suggest in your ServiceProvider or Routes file ([example](https://github.com/rapidez/core/blob/aa1dbb54faed244b982f5b6198749ccf493c210a/src/RapidezServiceProvider.php#L87)).
 
 The first argument to this function will be your Callable or action, the second argument will be the position or priority it has. Lower means higher priority, but it is optional.
 
 ::: tip
-If your check to see if it matches has a high performance impact (for example when it's an API request), consider putting the position higher than `9999` and caching the results.
+If your check to see if it matches has a high-performance impact (for example when it's an API request), consider putting the position higher than `9999` and caching the results.
```

Comment:
```suggestion
If your check to see if it matches has a high performance impact (for example when it's an API request), consider putting the position higher than `9999` and caching the results.
```

The "impact" isn't what's high-performance, the "performance impact" is what's high